lib/scsi_defs.h

```diff
@@ -97,5 +99,77 @@ namespace scsi_defs {
   } ABSL_ATTRIBUTE_PACKED;
   static_assert(sizeof(InquiryCommand) == kInquiryCommandSize);
 
+
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 3.6.2 Table 60
+  enum class PeripheralQualifier: uint8_t {
+    kPeripheralDeviceConnected = 0b000,
+    kPeripheralDeviceNotConnected = 0b001,
+    kReserved = 0b010,
+    kNotSupported = 0b011
+  };
+
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 3.6.2 Table 63
+  enum class TPGS : uint8_t {
+    kNotSupported = 0b00,
+    kImplicitAccess = 0b01,
+    kExcplicitAccess = 0b10,
+    kFullAccess = 0b11,
+  };
+
+  enum class ResponseDataFormat : uint8_t {
```

Comment:
Can we add a reference to this?

Reply:
There's actually no reference to this in the Seagate documentation. Its there on the internal documentation that we have. Not sure how to provide a reference in this case